### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by leopoldev404** [orafce/orafce#283]
![image](https://github.com/user-attachments/assets/57a89095-df63-4a83-81c2-7eead517ceaa)

when running `make` inside the main folder

**Comment by okbob** [comment]
út 8. 10. 2024 v 15:01 odesílatel Leonardo Pol ***@***.***>
napsal:

> image.png (view on web)
> <https://github.com/user-attachments/assets/57a89095-df63-4a83-81c2-7eead517ceaa>
>
> when running make inside the main folder
>

what is version of Postresql?

**Comment by leopoldev404** [comment]
postgres is version 9.6

**Comment by leopoldev404** [comment]
I have another postgresql db setup with orafce, I am trying to replicare that postgres on a docker container in order to perform some tests without affecting the real db

**Comment by okbob** [comment]
út 8. 10. 2024 v 23:47 odesílatel Leonardo Pol ***@***.***>
napsal:

> I have another postgresql db setup with orafce, I am trying to replicate
> that postgres on a docker container in order to perform some tests without
> affecting the real db
>

you cannot to use current Orafce against PostgreSQL 9.6. It should not work

last Orafce with PostgreSQL 9.6 support is 3.25.1

Minimally this commit breaks support for 9.6
https://github.com/orafce/orafce/commit/4f873e9f37fd32bd76e4e23a9b0473c1a9d46f05

and there is lot of new code, that was not newer tested against 9.6 and
compiled.

PostgreSQL 9.6 is unsupported now
https://www.postgresql.org/support/versioning/ - the support of 9.6
ended November 11, 2021

**Comment by leopoldev404** [comment]
yeah I know postgres 9.6 is not supported anymore, but for business requirements I need it.
Which orafce version should I use to make it work with postgres 9.6?

**Comment by leopoldev404** [comment]
thank you so much, using npgxclient with the right version did the trick!

**Comment by okbob** [comment]
st 9. 10. 2024 v 9:57 odesílatel Leonardo Pol ***@***.***>
napsal:

> thank you so much, using npgxclient with the right version did the trick!
>
:-)
